packages/re6stnet.rb

```diff
@@ -24,16 +24,16 @@ def self.build
   end
 
   def self.install
-    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/opt/ "
+    system "mkdir -p #{CREW_DEST_PREFIX}/opt/ "
     system "cp -r #{@re6st_dir} #{CREW_DEST_DIR}#{@re6st_dir}"
     system "rm -rf #{CREW_DEST_DIR}#{@re6st_dir}/download-cache"
+    system "ln -fs #{@grande_dir}/bin/grandenet #{CREW_PREFIX}/sbin/grandenet"
```

Comment:
Where did you define `@grande_dir`?

Reply:
whoops sorry, a search and replace gone wrong